fulp_modules/features/antagonists/bloodsuckers/code/bloodsucker/bloodsucker_recipes.dm

```diff
@@ -10,7 +10,7 @@
 		/obj/item/stack/sheet/iron = 1,
 	)
 	time = 15 SECONDS
-	category = CAT_PRIMAL
+	category = CAT_FURNITURE
```

Comment:
Set to ``CAT_STRUCTURE`` please, that's what I did in my currently open PR and they aren't furniture.
I'll revert the change on my own PR